From 0724904b78101456b47a8109dec539ea7f997a63 Mon Sep 17 00:00:00 2001
From: graysky <therealgraysky AT proton DOT me>
Date: Mon, 20 Jan 2025 16:45:58 -0500
Subject: [PATCH] f2fs_io.h: add missing flags

---
 tools/f2fs_io/f2fs_io.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/tools/f2fs_io/f2fs_io.c b/tools/f2fs_io/f2fs_io.c
index fa01f8fd4809..c2cc7f8634bd 100644
--- a/tools/f2fs_io/f2fs_io.c
+++ b/tools/f2fs_io/f2fs_io.c
@@ -43,6 +43,17 @@
 #include <android_config.h>
 
 #include "f2fs_io.h"
+#ifndef FALLOC_FL_COLLAPSE_RANGE
+#define FALLOC_FL_COLLAPSE_RANGE    0x08
+#endif
+
+#ifndef FALLOC_FL_INSERT_RANGE
+#define FALLOC_FL_INSERT_RANGE      0x20
+#endif
+
+#ifndef FALLOC_FL_ZERO_RANGE
+#define FALLOC_FL_ZERO_RANGE        0x10
+#endif
 
 struct cmd_desc {
 	const char *cmd_name;
-- 
2.48.1

